internal/rbd/errors/errors.go

```diff
@@ -55,4 +63,8 @@ var (
 	ErrInvalidArgument = errors.New("invalid arguments provided")
 	// ErrImageInUse is returned when the image is in use.
 	ErrImageInUse = errors.New("image is in use")
+	// ErrRBDGroupNotConnected is returned when the RBD group is not connected.
+	ErrRBDGroupNotConnected = fmt.Errorf("%w: RBD group is not connected", librados.ErrNotConnected)
```

Comment:
This is in a RBD sub-package, drop the RBD part of all variables, make it `ErrGroupNotConnected`.